src/routes/github/create-branch/github-create-branch-options-get.ts

```diff
@@ -71,8 +71,7 @@ const getJiraHostFromTenantUrl = (jiraHostParam): string | undefined =>  {
 	if (!jiraHostParam) {
 		return undefined;
 	}
-	const siteName = jiraHostParam?.substring(0, jiraHostParam?.indexOf("."));
-	return `https://${siteName}.atlassian.net`;
```

Comment:
The correct domain is coming from this variable, we need to keep what it was not convert to .atlassian.net